### PR TITLE
Change /etc/openshift-passwd to /etc/openshift/htpasswd

### LINKED ISSUE #130: put htpasswd file in /etc/openshift

### PR DESCRIPTION
**Pull request by dobbymoodge**
Fixes #130

### CODE REVIEW

**Comment by dobbymoodge** [comment]
@thoraxe

File: beta-4-setup.md [hunk 1]
```diff
@@ -412,7 +412,7 @@ If you're feeling lazy, use your friend `sed`:
 
     sed -i -e 's/name: anypassword/name: apache_auth/' \
     -e 's/kind: AllowAllPasswordIdentityProvider/kind: HTPasswdPasswordIdentityProvider/' \
-    -e '/kind: HTPasswdPasswordIdentityProvider/i \      file: \/etc\/openshift-passwd' \
+    -e '/kind: HTPasswdPasswordIdentityProvider/i \      file: \/etc\/openshift/htpasswd' \
```
Comment (brenton) [inline beta-4-setup.md:415]: Actually, are we certain there aren't going to be escaping issues here with `file: \/etc\/openshift/htpasswd`?

Comment (dobbymoodge) [inline beta-4-setup.md:415]: @brenton yup, I'll commit another PR. Good eye.

**Comment by thoraxe** [comment]
I've not seen another PR for this?

**Comment by thoraxe** [comment]
@dobbymoodge

**Comment by dobbymoodge** [comment]
@thoraxe Hmm, just saw this notification, sorry. The followup PR was #134

**Pull request merged**